assets/js/script.js

```diff
@@ -67,11 +67,81 @@ $(document).ready(function () {
 
         return (ua.indexOf('FBAN') > -1) || (ua.indexOf('FBAV') > -1);
     }
-
+  
     if (isFacebookBrowser()) {
+      
+        $('#facebook-browser').removeClass("d-none");
+        $('#allow-page-load').addClass("d-none");
+    }
+
+    const localStorage_save = "save-locally";
+    const localStorage_data = "form-data";
+
+    let saveLocallyEnabled = localStorage.getItem(localStorage_save) === "false" ? false : true;
+
+    const saveIgnoredItems = ["form_start_hour",  "form_start_minute", "form_end_hour", "form_end_minute", "form_date"];
```

Comment:
Salutare @kriskate si multumim pentru contributie.

In ultima versiune a formularului, campurile "form_start_hour",  "form_start_minute", "form_end_hour", "form_end_minute" nu mai exista. As sugera sa le scoti si de aici.